### PR TITLE
Fix popup player resize

### PR DESCRIPTION
**Pull request by nv95**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

Made resizing of popup player window relative

### CODE REVIEW

**Comment by Gymcap** [comment]
Also allow pinch/spread resize from any orientation. Currently this is horizontal only according to my observations.

**Comment by nv95** [comment]
@Gymcap what you mean as "horizontal only"?

**Comment by gkeegan** [comment]
@nv96 start a player in popup player, then try to adjust the sizing only with pinches in the vertical direction. It causes the player to shrink to the minimum size and then doesn't adjust back without a horizontal pinch.

**Comment by Gymcap** [comment]
@nv95 what gkeegan said

File: app/src/main/java/org/schabi/newpipe/player/PopupVideoPlayer.java [hunk 1]
```diff
@@ -296,9 +298,7 @@ private void resetNotification() {
                 .setSmallIcon(R.drawable.ic_newpipe_triangle_white)
                 .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
                 .setContent(notRemoteView);
-        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.JELLY_BEAN) {
```
Comment (TobiGr) [inline app/src/main/java/org/schabi/newpipe/player/PopupVideoPlayer.java]: CC @friendlyanon

Comment (friendlyanon) [reply app/src/main/java/org/schabi/newpipe/player/PopupVideoPlayer.java]: Thank you for the mention. Luckily JB is API 16, so this is not an issue :)

**Comment by nv95** [comment]
@gkeegan I cannot reproduce it(
It works fine for me

**Comment by Stypox** [comment]
Test apk: 
[app-debug.zip](https://github.com/TeamNewPipe/NewPipe/files/3511953/app-debug.zip)

**Comment by Gymcap** [comment]
@nv95  Streamable ate the quality, but [this video](https://streamable.com/na9p6) should help clarify the behaviour we're talking about. 

Notice the two white dots showing where my fingers are. When I attempt to adjust the player size with my fingers horizontal, it works fine. If my fingers are vertical however, the player thinks I'm asking for the smallest size.

**Comment by nv95** [comment]
@Gymcap thanks. But I have an another behavior on my device...
I will try to reproduce it again on another device one Android version

**Comment by haistakaavittukaikkisaatana** [comment]
Im not sure if this a bug or how its supposed to work, but I can resize popup window with only one finger, by holding my finger on top of the popup window a half a second will resize the popup window. I dont mind at all if this is a bug, I kinda like it for simplicity lol.

Check this video:

https://streamable.com/s/b7jw8

**Pull request closed without merging**